src/test/java/com/stripe/StripeTest.java

```diff
@@ -2072,67 +2070,7 @@ public void testAlipayAccountDeleting() throws StripeException {
 		AlipayAccount alipayAccount = (AlipayAccount) cus.getSources().getData().get(0);
 		alipayAccount.delete();
 
-		PaymentSourceCollection sources = cus.getSources().all(new HashMap<String, Object>());
+		ExternalAccountCollection sources = cus.getSources().all(new HashMap<String, Object>());
 		assertEquals(0, sources.getData().size());
 	}
-
-	@Test
-	public void testAlipayAccountAsUnknownSourceCreation() throws StripeException {
```

Comment:
It really would be great to have a test of gracefully handling an unknown external account type.
If there's no good way to hack a mapping anymore (I see why it wouldn't be straightforward to do that), maybe it's better to switch these tests over to using network stubbing?